src/test/java/gov/cms/mat/cql_elm_translation/service/CqlConversionServiceTest.java

```diff
@@ -182,6 +182,9 @@ void testProcessCqlDataWithErrorsMissingModel() {
   @Test
   void testGetTranslatedLibrariesForCqlForCql() throws IOException {
     String cql = getData("/qdm_data_criteria_retrieval_test.cql");
+    MadieLibrarySourceProvider.setUsing(new CqlTextParser(cql).getUsing());
+    MadieLibrarySourceProvider.setCqlLibraryService(cqlLibraryService);
+    MadieLibrarySourceProvider.setAccessToken("access token");
```

Comment:
this is unrelated to this ticket. I will revert if anyone objects. this is to fix the gundam build failure: https://qnetjenkins.cms.gov/bmat-master/job/Madie/job/Madie%20Services/job/cql-elm-translator-service/job/sprint%252Fgundam_2024-01-29/3/console.